pages/mensa.vue

```diff
@@ -39,7 +46,27 @@
       </v-flex>
     </v-layout> 
     <span class="disclaimer">
-      Quelle: openmensa.org
+      <span>
+        Vegetarisch
+        <v-icon
+          :color="getIconColor()"
```

Comment:
Schwarz wirkt da irgendwie out of place... vlt. mit hellem oder dunklen Grün?

Reply:
es gibt doch eine Legende, ganz unten rechts wo openmensa.org steht